server/lobbyconnection.py

```diff
@@ -419,34 +445,34 @@ def _set_user_agent_and_version(self, user_agent, version):
     async def _check_version(self):
         versionDB, updateFile = self.player_service.client_version_info
         update_msg = {
-            'command': 'update',
-            'update': updateFile,
-            'new_version': versionDB
+            "command": "update",
+            "update": updateFile,
+            "new_version": versionDB
         }
 
-        if not self.user_agent or 'downlords-faf-client' not in self.user_agent:
+        if not self.user_agent or "downlords-faf-client' not in self.user_agent:
             await self.send_warning(
                 "You are using an unofficial client version! "
                 "Some features might not work as expected. "
                 "If you experience any problems please download the latest "
                 "version of the official client from "
-                f'<a href="{config.WWW_URL}">{config.WWW_URL}</a>'
+                f"<a href=\"{config.WWW_URL}\">{config.WWW_URL}</a>"
```

Comment:
And here's a situation where I think single quotes are appropriate in order to avoid backslash escapes.